Examples/test-suite/cpp11_shared_ptr_director.i

```diff
@@ -0,0 +1,90 @@
+%module(directors="1") "cpp11_shared_ptr_director"
+
+%{
+#include <memory>
+%}
+
+%include "std_shared_ptr.i";
```

Comment:
Please test using C++98 and boost_shared_ptr.i instead of std_shared_ptr.i as std::shared_ptr cannot currently be tested for all the languages that support shared_ptr in boost as these languages are not C++11 ready yet. Extending the typemaps to all the other languages which have boost_shared_ptr.i would be my ultimate goal, so better to set it up ready for the additions to the other target languages; replicating the typemap changes in all the other target languages won't be too hard.